package.json

```diff
@@ -9,6 +9,7 @@
     "packages/monitor",
     "packages/multi-provider",
     "packages/local-environment",
+    "packages/manual-processor",
```

Comment:
this shouldn't be its own package - we can consider having a package for various chain ops scripts (this is something that's come up repeatedly)